README.md

```diff
@@ -141,13 +141,36 @@ The service will respond to requests on `localhost:2424`, and the admin data wil
 
 ### Configuration
 
-Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper).
-The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file.
+Configuration is handled by [Viper](https://github.com/spf13/viper#putting-values-into-viper). The easiest way to set config during development is by editing the [config.yaml](./config.yaml) file. You can also set the config through environment variables. For instance:
 
-You can also set the config through environment variables. For example:
+```bash
+PBC_COMPRESSION_TYPE: "none"
+```
+
+Prebid Cache's rate limiting feature is also configurable. By default, it is enabled and allows for a maximum of 100 requests per second. Given its high memory consumption, it can be disabled by setting the `rate_limiter.enabled` flag to `false` in the configuration file [config.yaml](./config.yaml) or the `PBC_RATE_LIMITER_ENABLED` environment variable. This is what a sample configuration that disables the rate limiter in the [config.yaml](./config.yaml) file looks like:
```

Comment:
In the first sentence, please consider mentioning that Prebid Cache has a rate limiting feature enabled by default that uses a large amount of unbounded memory.